packages/suite/src/views/wallet/transactions/components/TransactionList/components/Pagination/index.tsx

```diff
@@ -11,10 +11,10 @@ const Wrapper = styled.div`
 const PageItem = styled.div<{ isActive?: boolean }>`
     cursor: pointer;
     font-size: ${variables.FONT_SIZE.SMALL};
-    background: ${props => (props.isActive ? colors.GREEN : 'transparent')};
-    color: ${props => (props.isActive ? colors.WHITE : colors.GREEN)};
+    background: ${props => (props.isActive ? props.theme.BG_GREEN : 'transparent')};
+    color: ${props => (props.isActive ? props.theme.TYPE_WHITE : props.theme.TYPE_GREEN)};
     padding: 4px 8px;
-    border-radius: 2px;
+    border-radius: 2px;s
```

Comment:
here is some left behind 's'.
this is actually something I don't like very much about styled components, you may miss `;` and checks still pass

Reply:
good thing we have good reviewers :) I thought that would be caught by some of our linters, weird.